examples/browser/counter.js

```diff
@@ -113,7 +113,7 @@ function showInteractions(thing) {
                     eventSubscriptions[evnt] = true;
                     thing
                         .subscribeEvent(evnt, async function (data) {
-                            console.log("Data:" + (await data.value()));
+                            console.log('Event "' + evnt + '"');
```

Comment:
This should output just the `InteractionOutput` object, is it really what we want? In principle the previous code was correct and was showing how to properly handle event payloads. I think this new code it is a little bit misleading for new users. Another option is to add as payload  `null` so that the example will still work and you can use `await data.value()` here.

Reply:
Yes, either we report null OR we leave the console log as is.. it does only print the event name.

Reply:
Oh sorry, I've overlooked I was thinking that you were printing `data`. Ok for merging then!